onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -326,6 +326,7 @@ will be calculated.)DOC";
       .SinceVersion(1)
       .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
       .SetDoc(EmbedLayerNormalization_ver1_doc)
+      .Attr("epsilon", "The epsilon value to use to avoid division by zero.", AttributeProto::FLOAT, 1e-13f)
```

Comment:
Use same default value of **1e-12** as SkipLayerNormalization.

Reply:
Is there any reason why the values are varies by ops in the first place? I'm asking because the unit tests for SkipLayerNormalization seem to be built based on 1e-12f. If I change it to 1e-5, the tests will fail.

Reply:
Then probably we can keep different default values of each op, and update fusion logic to always set epsilon value for fused node. That is good for backward compatible as well.